### ISSUE TITLE
High Sierra working in degraded fashion as a window.

### ISSUE THREAD

**Issue by MichaelDimmitt** [MichaelDimmitt/Aerial_Desktop#38]
Currently Implemented, you can check out branch [macintosh_version_checking](https://github.com/MichaelDimmitt/Aerial_Desktop/tree/macintosh_version_checking)
This weekend I will attach an executable app for non programmers to download the updated Aerial Desktop.

Previous to High Sierra Work.  
High Sierra working in degraded fashion as a window.

[github.com/MichaelDimmitt/Aerial_Desktop/pull/37](https://github.com/MichaelDimmitt/Aerial_Desktop/pull/37)

The pull request will be merged into master when 3 things are achieved:

0) ~Backdoor, putting computer to sleep twice in less than two minutes shuts off launch agent.~
0) ~High Sierra working in degraded fashion~ #38
1) Application will prompt user if it finds a new release than project currently displays. #39 
2) Add Install window allow user to choose normal or menubar implementation. #40

![degraded_high_sierra_example](https://user-images.githubusercontent.com/11463275/41569503-77a9c12c-7339-11e8-8f52-f1f29f7ca935.png)

**Comment by dougluce** [comment]
Hey @MichaelDimmitt, thanks for tackling this. I've checked out the branch (at `abc6471a`) on my 10.13.6 box. After doing the `open .hide/bin/Installer.app` I get the notice "Installed for this user only, enjoy."

However, my desktop still shows the stock static background. No window pops up or anything. Any advice?

**Comment by MichaelDimmitt** [comment]
@dougluce, if I understand your background did not change.

Static is your default desktop background?

Im looking into it. 
And I should have pushed this code a while back. I Did not know anyone would want the degraded version. 

The fix should be a quick one. When we get it working on your computer let me know  your thoughts.

**Comment by dougluce** [comment]
Yup, my background didn't change from the image that had been on there 
since the OS was installed. If you'd like me to do any sort of diagnosis, 
please let me know what.

**Comment by MichaelDimmitt** [comment]
I never pushed the release.

@dougluce, I will push it and notify you right after I test it out on my end.
I have to go find a mac with high sierra; mine is still using sierra. :). 

I will post two releases hopefully after two hours. 

The Aerial_Desktop enabled with the High_Sierra 
and a release that has the same changes with a Mac Menubar as the app to run it.

**Comment by MichaelDimmitt** [comment]
@dougluce, pushed a new release for the degraded version. 
Try it out, verified working on a Mac Book Air, OS: macOS HighSierra 10.13.6

## Instructions: to download the new release

1) [Click here to Download](https://github.com/MichaelDimmitt/Aerial_Desktop/releases/download/v9.1/Aerial_Desktop.zip)
2) Unzip the downloaded file.
3) hold "ctrl" and Open **Installer** and confirm installation.<br>

I am not releasing the menubar tonight.
Let me know if the degraded version is working for you now. 

Thanks, for the feedback. And the motivation to knock the feature out. 
changes were implemented on this branch:
https://github.com/MichaelDimmitt/Aerial_Desktop/tree/CurrentStableRelease

**Comment by dougluce** [comment]
That gets closer! Here's ended up on the external display that I regularly use as my main monitor:

![dell](https://user-images.githubusercontent.com/1047838/45836296-48269c00-bcc1-11e8-99f3-45b985e94e8c.png)

The window is (of course) non-selectable and non-resizable, but is animated and appears underneath everything.

I also tried on the Macbook's built-in display. This is what it looks like (after running the Stop and Installer apps again):

<img width="1440" alt="retina" src="https://user-images.githubusercontent.com/1047838/45836295-48269c00-bcc1-11e8-9e2f-d56b0e4ec1a3.png">
